.github/workflows/embedded.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           EMBED_BUILD_LLVM: OFF
           EMBED_USE_LLVM: ON
           RUN_ALL_TESTS: 1
+          RUN_MEMLEAK_TEST: 1
```

Comment:
Should we build release assets with ASAN?

Reply:
The way it's done now is that two binaries are built: *bpftrace* and *bpftrace-asan*. We could possibly run `make clean` after the memleak tests which should remove unnecessary files built with ASAN.

Reply:
I only see one build [here](https://github.com/iovisor/bpftrace/actions/runs/1026678998) (for example, `bpftrace-Release-vanilla_llvm+clang+glibc2.27`, which is built with ASAN I guess), or am I missing? I think it's OK to publish two binaries unless it costs too much (e.g., increasing publish time largely or hitting storage limits or something.)

Reply:
Those artifacts just package the *bpftrace* binary which is built without ASAN (see [here](https://github.com/iovisor/bpftrace/blob/master/.github/workflows/embedded.yml#L147-L154)) so there should be no problem, I believe.

Reply:
ah, I missed that `RUN_MEMLEAK_TEST` let `build.sh` build not only `bpftrace` but also `bpftrace-asan`. Then it seems fine.